qiita_pet/portal.py

```diff
@@ -0,0 +1,87 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+from future import standard_library
+from os.path import join, dirname, abspath
+
+from qiita_core.exceptions import MissingConfigSection
+from qiita_core.qiita_settings import qiita_config
+
+with standard_library.hooks():
+    from configparser import ConfigParser
+
+
+class PortalStyleManager(object):
+    """Holds the portal style information
+
+    Parameters
+    ----------
+    conf_fp: str, optional
+        Filepath to the configuration file. Default: portal.txt
+
+    Attributes
+    ----------
+    logo : str
+        Path from base URL to the site logo
+    title : str
+        Site title
+    index header : str
+        Welcome text header for the website
+    index_text : str
+        Welcome text for the website
+    example_search : str
+        Example search to be shown on the study listing page
+    custom_css : str
+        custom css for the portal
+    conf_fp : str
+        The filepath to the portal styling config file
+    css_fp : str
+        The filepath to the portal styling custom CSS
+    """
+    def __init__(self, conf_fp=None):
```

Comment:
Looks like the `conf_fp` parameter is ignored (probably leftover code). Can you remove?